css/index.css

```diff
@@ -112,6 +128,46 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes .box-1{
```

Comment:
For these boxes, you have a class name applied to each box, so you do not need to have .middle-content .boxes on each of these. 

Another little thing, since CSS stands for Cascading Style Sheet we know it cascades down.  But it also uses specificity.  If you were to specify .middle-content .boxes. box-1 { background-color: black }  then below it specify .box-1 {background-color: red; } which color would it be?  It would be black, because that is more specific (in most browsers).